x/oracle/module.go

```diff
@@ -169,3 +173,54 @@ func (am AppModule) EndBlock(ctx sdk.Context, _ abci.RequestEndBlock) []abci.Val
 
 	return []abci.ValidatorUpdate{}
 }
+
+// AppModuleSim implements the AppModule interface for the x/oracle module.
+type AppModuleSim struct {
```

Comment:
Had to add the `accountKeeper` and `bankKeeper` for weighted operations but didn't want to touch `AppModule` - is this the appropriate way to do this? Or should we just use `AppModule` and add the `accountKeeper` and `bankKeeper` to it 
cc : @alexanderbez

Reply:
Answer to this will be relevant to @RafilxTenfen 's PR

Reply:
So yes, typically you add these keeper dependencies in `AppModule`, which you then pass those types to the relevant simulation logic/calls.

That being said, you can also define a special `AppModuleSim`, but AFAIK, that isn't the standard in the SDK. So if we're going with consistency, you can just have them set on `AppModule`. But I don't have a strong preference either way. I just wanted be consistent _within umee_.

Reply:
ref: https://github.com/umee-network/umee/pull/401#discussion_r787990311